config/studio/content-types/page/entry/form-definition.xml

```diff
@@ -451,7 +451,7 @@
 						</property>
 						<property>
 							<name>contentTypes</name>
-							<value>/component/component-catalog,/component/component-newsletter,/component/component-overview,/component/component-quotes,/component/component-service,/component/component-trainers</value>
+							<value>/component/component-catalog,/component/component-newsletter,/component/component-overview,/component/component-quotes,/component/component-trainers</value>
```

Comment:
Issue with service component is that it was not supposed to be allowed in the entry 'Page Sections' field, since it is a component with no template used by catalog item component